CONTRIBUTING.md

```diff
@@ -132,6 +132,36 @@ Please remember to:
 * Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
 
 
+```mermaid
+timeline
+    title Code integration journey (CI)
+    Project setup <br> (make dev)   : Code checkout
+                                    : Virtual environment
+                                    : Dependencies
+                                    : Git pre-commit hooks
+                                    : Local branch
+                                    : Local changes
+                                    : Local tests
+
+    Pre-Pull Request <br> (make PR)     : Code linting
+                                        : Docs linting
+                                        : Static typing analysis
+                                        : Tests (unit|integration)
```

Comment:
What's the difference between local tests and Test (unit) ?

Reply:
This would be running `tox` commands whereas local testing is functional testing